virtme_ng/run.py

```diff
@@ -256,6 +255,12 @@ def make_parser():
         + "(useful with --arch)",
     )
 
+    parser.add_argument(
```

Comment:
(detail: maybe best to move this options elsewhere, not in the middle of the "filesystems" options, at least not between `--root` and `--root-release` ; but not sure if there is really a good spot :) maybe at the end, maybe we should re-organise the options)